Game/Src/TestEnemyMovement.gd

```diff
@@ -46,7 +45,7 @@ var base_pixel_speed
 var curr_pixel_speed
 var curr_scroll_speed
 var init_child_num
-var rng = RandomNumberGenerator.new()
+onready var rng = RandomNumberGenerator.new()
```

Comment:
this seems unused now

Reply:
This is still used for random enemy spawn